lang/c/c.talon

```diff
@@ -58,35 +64,45 @@ state default: "default:\nbreak;"
 #best used with a push like command
 #the below example may not work in editors that automatically add the closing brace
 #if so uncomment the two lines and comment out the rest accordingly
-push braces:
+push (brace|braces):
     edit.line_end()
-    #insert("{")
-    #key(enter)
-    insert("{}")
-    edit.left()
+    insert("{")
     key(enter)
-    key(enter)
-    edit.up()
+    # insert("{}")
+    # edit.left()
+    # key(enter)
+    # key(enter)
+    # edit.up()
+
+push (semi|semicolon):
+     edit.line_end()
+     insert(";")
+     key(enter)
 
 # Declare variables or structs etc.
 # Ex. * int myList
-<user.c_variable> <phrase>:
+(var|variable) <user.c_variable> <phrase>:
```

Comment:
Would recommend keeping track of any places where you are changing existing functionality such it can be added to the breaking changes file.

Reply:
I thought about something like this, but then I basically completely overhauled the mode. I really wish I knew someone who was actually using the current C mode (or was confident that there actually isn't anyone).

Reply:
There are definitely people using the existing C mode. If you add things that are C++ only, no need to document that, only if you change them. This is the standard we used for community maintenance.